emcc.py

```diff
@@ -2003,6 +2003,7 @@ def phase_linker_setup(options, state, newargs):
       'emscripten_stack_get_free',
       'emscripten_stack_get_base'
     ]
+    settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$ptrToString']
```

Comment:
How does this function get included in non-`STACK_OVERFLOW_CHECK` builds? I'm not sure how all the new uses added in this PR ensure that the function is included.

Reply:
I think because all the other uses are behind ASSERTIONS or DEBUG macros, and ASSERTIONS enables STACK_OVERFLOW_CHECK.  I can add more dependencies.